helpdesk/templates/helpdesk/kb_index.html

```diff
@@ -5,17 +5,23 @@
 
 <p>{% trans "We have listed a number of knowledgebase articles for your perusal in the following categories. Please check to see if any of these articles address your problem prior to opening a support ticket." %}</p>
 
-<table class="table table-hover table-bordered table-striped">
-<caption>{% trans "Knowledgebase Categories" %}</caption>
-<thead>
-<tr><th>{% trans "Category" %}</th></tr>
-</thead>
-<tbody>
 {% for category in kb_categories %}
-<tr><th><a href='{{ category.get_absolute_url }}'>{{ category.title }}</a></th></tr>
-<tr><td>{{ category.description }}</td></tr>
+{% cycle 'one' 'two' 'three' as catnumperrow silent %}
+{% ifequal catnumperrow 'one' %}<div class="row">{% endifequal %}
+    <div class="col-lg-4">
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                <h3>{{ category.title }}</h3>
+            </div>
+            <div class="panel-body">
+                <p>{{ category.description }}</p>
+            </div>
+            <div class="panel-footer">
+                View <a href='{{ category.get_absolute_url }}'>{{ category.title }} articles <i class="fa fa-arrow-right"></i></a>
```

Comment:
Need a blocktrans tag here to wrap this up into a translator-friendly-style "View (%s) articles".